services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/service/impl/DpmServiceImpl.java

```diff
@@ -108,10 +117,10 @@ private UnicastGoalState buildUnicastGoalState(NetworkConfiguration networkConfi
                     networkConfig, hostIp, portEntities, multicastGoalState));
         }
 
-        return dataPlaneClient.createGoalStates(unicastGoalStates, multicastGoalState);
+        return dataPlaneClient.sendGoalStates(unicastGoalStates, multicastGoalState);
     }
 
-    private List<String> createPortConfiguration(NetworkConfiguration networkConfig) throws Exception {
+    private List<String> processPortConfiguration(NetworkConfiguration networkConfig) throws Exception {
```

Comment:
Could you add brief comments to this method?

Reply:
No problem